src/Interpreters/ClusterProxy/SelectStreamFactory.cpp

```diff
@@ -131,7 +131,7 @@ void SelectStreamFactory::createForShard(
             return;
         }
 
-        UInt32 local_delay = replicated_storage->getAbsoluteDelay();
+        UInt64 local_delay = replicated_storage->getAbsoluteDelay();
```

Comment:
`getAbsoluteDelay()` returns `time_t`.

Reply:
This is to compare it with `settings.max_replica_delay_for_distributed_queries`